eng/scripts/Fetch-PackageList-Javadoc.ps1

```diff
@@ -0,0 +1,51 @@
+<#
+  .SYNOPSIS
+  Fetch the package list from javadoc jar and checkin to docs/metadata
+
+  .DESCRIPTION
+  The Verify-Links.ps1 script will check whether the files contain any broken links.
+
+  .PARAMETER JavaDocJarLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER DocRepoLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER ArtifactName
+  Specifies the file that contains a set of links to ignore when verifying.
+#>
+[CmdletBinding()]
+param (
+  [string] $JavaDocJarLocation = "",
+  [string] $DocRepoLocation = "",
+  [string] $ArtifactName = ""
+)
+. (Join-Path $PSScriptRoot ../common/scripts/common.ps1)
+
+function Fetch-Namespaces-From-Javadoc ($jarFilePath, $tempLocation, $destination) {
+  Push-Location $tempLocation
+  jar xf $jarFilePath
+  if (Test-Path "./element-list") {
+      # Rename and move to location
+      Copy-Item "./element-list" -Destination "$packageNameLocation/$ArtifactName.txt"
+      Write-Host "Copying the element-list to $packageNameLocation/$ArtifactName.txt"
+  }
+  Pop-Location
+}
+Write-Host "The artifact name: $ArtifactName"
+$jarFile = Get-ChildItem $JavaDocJarLocation -Recurse -Include "$ArtifactName*-javadoc.jar"
+Write-Host "The jar file is $jarFile."
+$version = $jarFile.Name -replace "$ArtifactName-(.*)-javadoc.jar", '$1'
+Write-Host "The full version: $version"
+$tempLocation = Join-Path ([System.IO.Path]::GetTempPath()) "jarFiles"
```

Comment:
This should be a parameter to the script with this location as the default value.

Reply:
Prefer to add necessary parameters only as newly added parameters do not treat as breaking changes. But update the existing parameter name is breaking change.